.github/workflows/version-bump.yml

```diff
@@ -0,0 +1,67 @@
+---
+name: Version Bump
+
+on:
+  workflow_dispatch:
+    inputs:
+      version_number:
+        description: "New Version"
+        required: true
+
+jobs:
+  bump_props_version:
+    name: "Create version_bump_${{ github.event.inputs.version_number }} branch"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create Branch
+        uses: peterjgrainger/action-create-branch@v2.0.1
```

Comment:
I feel more comfortable minimizing the usage of third-party actions as much as possible to limit supply chain attack vectors in our CI pipelines. This can easily be done using the git command line after the repo has been checked out. See the [web release](https://github.com/bitwarden/web/blob/221931ecaa764b91929eef9ab8a9ab703b5ce274/.github/workflows/release.yml#L97) as reference